src/components/orator/js/Orator.js

```diff
@@ -847,6 +856,10 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
             stop: {
                 funcName: "fluid.orator.selectionReader.stopSpeech",
                 args: ["{that}.model.play", "{fluid.textToSpeech}.cancel"]
+            },
+            toggle: {
+                funcName: "fluid.orator.selectionReader.togglePlay",
```

Comment:
Understand the `state` argument is implicitly passed in via jQuery click event. But it would be more readable and reusable to include the pass-in of `{arguments}.0` on the argument list.

The same comment applies to the other `toggle()` function at line 161.